js/pom.xml

```diff
@@ -27,8 +27,11 @@
     <properties>
         <node.version>v22.11.0</node.version>
         <pnpm.version>9.14.4</pnpm.version>
-        <!-- The JavaScript projects use the non-standard 'src' folder for their sources, therefore, name it here explicitly -->
+        <!-- The JavaScript projects use non-standard folders for their sources, therefore, name it here explicitly -->
```

Comment:
@ahus1 why do we need to configure the Maven build cache with these directories when the JS build completely circumvents the Maven build?

Reply:
Reading your comment here https://github.com/keycloak/keycloak/pull/35670#pullrequestreview-2484457963 now. Consider this resolved.

Reply:
See below: 

> The maven.build.cache.input.* properties are still needed for the Maven cache extension to register the changes that trigger the packaging of the respective JARs.

Without registering the source folders, the Maven cache extension would never re-package the JARs for the admin UI and account console.

Reply:
Yes, the commit and the comment were overlapping. ASYNC everywhere!